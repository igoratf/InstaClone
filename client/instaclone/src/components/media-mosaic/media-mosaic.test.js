import React from 'react';
import { shallow } from 'enzyme';
import MediaMosaic from './media-mosaic.component';

describe('Media mosaic component', () => {
  it('should render media mosaic component', () => {
    const mockProps = {profilePosts: []};
    expect(shallow(<MediaMosaic {...mockProps}/>)).toMatchSnapshot();
  })
})