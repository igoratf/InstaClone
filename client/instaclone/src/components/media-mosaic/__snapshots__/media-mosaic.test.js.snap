// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Media mosaic component should render media mosaic component 1`] = `ShallowWrapper {}`;
